src/main/java/com/azure/cosmos/examples/analyticalcontainercrud/async/AnalyticalContainerCRUDQuickstartAsync.java

```diff
@@ -0,0 +1,214 @@
+            e.printStackTrace();
+            logger.error("Cosmos getStarted failed", e);
+        } finally {
+            logger.info("Closing the client");
+            p.shutdown();
+        }
+    }
+
+    private void containerCRUDDemo() throws Exception {
+
+        logger.info("Using Azure Cosmos DB endpoint: {}", AccountSettings.HOST);
+
+        // Create async client
+        client = new CosmosClientBuilder()
+                .endpoint(AccountSettings.HOST)
+                .key(AccountSettings.MASTER_KEY)
+                .consistencyLevel(ConsistencyLevel.EVENTUAL)
+                .contentResponseOnWriteEnabled(true)
+                .buildAsyncClient();
+
+        createDatabaseIfNotExists();
+        createContainerIfNotExists();
+        updateContainerThroughput();
+
+        readContainerById();
+        readAllContainers();
+
+        // deleteAContainer() is called at shutdown()
+
+    }
+
+    // Database Create
+    private void createDatabaseIfNotExists() throws Exception {
+        logger.info("Create database {} if not exists...", databaseName);
+
+        // Create database if not exists - this is async but we block to make sure
+        // database and containers are created before sample runs the CRUD operations on
+        // them
+        CosmosDatabaseResponse databaseResponse = client.createDatabaseIfNotExists(databaseName).block();
+        database = client.getDatabase(databaseResponse.getProperties().getId());
+
+        logger.info("Done.");
+    }
+
+    // Container create
+    private void createContainerIfNotExists() throws Exception {
+        logger.info("Create container " + containerName + " if not exists.");
+
+        // Create container if not exists
+        CosmosContainerProperties containerProperties = new CosmosContainerProperties(containerName, "/lastName");
+
+        // Set analytical store properties
+        containerProperties.setAnalyticalStoreTimeToLiveInSeconds(-1);
+
+        // Create container - this is async but we block to make sure
+        // database and containers are created before sample runs the CRUD operations on
+        // them
+        CosmosContainerResponse containerResponse = database.createContainerIfNotExists(containerProperties).block();
+        CosmosAsyncContainer container = database.getContainer(containerResponse.getProperties().getId());
+
+        logger.info("Done.");
+    }
+
+    // Update container throughput
+    private void updateContainerThroughput() throws Exception {
+        logger.info("Update throughput for container " + containerName + ".");
+
+        // Specify new throughput value
+        ThroughputProperties throughputProperties = ThroughputProperties.createManualThroughput(800);
+        container.replaceThroughput(throughputProperties).block();
+
+        logger.info("Done.");
+    }
+
+    // Container read
+    private void readContainerById() throws Exception {
+        logger.info("Read container " + containerName + " by ID.");
```

Comment:
same suggestion here, we can change to use: ` logger.info("Read container {} by ID", containerName.");`